test/generator/component-yield-multiple-in-if/_config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+	solo: true,
```

Comment:
Eek, good catch, thanks. I *think* `process.env.CI` is true in both Travis and Appveyor environments... let's find out! https://github.com/sveltejs/svelte/pull/244